src/config/serverConfigDefaults.ts

```diff
@@ -175,6 +175,8 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
     saml_logout_local: false,
     patient_view_use_legacy_timeline: false,
     enable_request_body_gzip_compression: false,
+
+    referenceGenomeVersion: 'hg19',
```

Comment:
@alisman still seems like a strange property since one portal can support multiple reference genomes - can we name this slightly differently? Like maybe more indicative that it's a cache key